src/screens/Dashboard/DeparturesWidget.tsx

```diff
@@ -120,6 +124,29 @@ const DeparturesWidget: React.FC = () => {
   );
 };
 
+function sortStopsByDistance(
```

Comment:
This function does not sort, so I think something like `compareStopsByDistance` would be better.